lib/components/link/link.less

```diff
@@ -133,4 +134,10 @@ a,
     p & {
         text-decoration: underline;
     }
+
```

Comment:
@dancormier This is so these interaction styles apply only for elements with the `s-link` class applied and not all `a` elements correct?

Reply:
Yep, the changes in this PR include a separate rule for `a` elements (lines 13-16 in this file) with the reference to the `.s-anchor` PPCP (`--_an-a-fc-hover`):

```less
        &:active,
        &:hover {
            color: var(--_an-a-fc-hover, var(--_li-fc-hover));
        } 
```

In contrast to `a`, we want styles applied to `.s-link` to override `.s-anchor a` styles, we'd want it to override any anchor styles, so I omitted the `.s-anchor` PPCP here.